projects/github.com/cowsay-org/cowsay/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: git+https://github.com/cowsay-org/cowsay.git
+  ref: ${{version.tag}}
```

Comment:
is it the case that using the packaged sources doesn't work? i'm fine if that's the case, but i'd be surprised for a perl package.

Reply:
Nope, updated.